testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -160,3 +161,44 @@ Feature: Smoke tests for <client>
     Then I should see a "1 package install has been scheduled for" text
     When I force picking pending events on "<client>" if necessary
     And I wait until event "Package Install/Upgrade scheduled by admin" is completed
+
+@skip_for_traditional
+  Scenario: Test Prometheus and Prometheus exporter formulas
+    Given I am on the Systems overview page of this "<client>"
+    When I follow "Formulas" in the content area
+    And I check the "prometheus" formula
+    And I check the "prometheus-exporters" formula
+    And I click on "Save"
+    Then I wait until I see "Formula saved" text
+    # Configure Prometheus formula
+    When I follow "Formulas" in the content area
+    And I follow "Prometheus" in the content area
+    And I click on "Expand All Sections"
+    And I enter "admin" as "Username"
+    And I enter "admin" as "Password"
```

Comment:
is this in the right place? :thinking: asking because I don't remember having had to write credentials at this part of doing this manually :thinking: just want to make sure it's been tested and confirmed

Reply:
When I test promtheus I use to pass the password, user it use to always be written, but anyway yes I guess it's fine.

Reply:
And btw, if you notice this is just an easy copy-paste of https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/secondary/min_monitoring.feature putting all together to make it easier to review in BV report, grouping feature in a unique scenario.

Reply:
Yes it needs credentials for monitoring the server